server/src/main/java/org/elasticsearch/transport/TransportService.java

```diff
@@ -654,7 +654,7 @@ public String toString() {
             if (ex instanceof TransportException) {
                 te = (TransportException) ex;
             } else {
-                te = new TransportException("failure to send", ex);
+                te = new TransportWrapperException("failure to send", ex);
```

Comment:
Looks right that we want to use a wrapper exception here, but we already have `SendRequestTransportException` and I think we should reuse it here for consistency and also because it carries slightly more information.

Reply:
Yes, much better than introduce a new Exception